packages/hadron-document/src/document.ts

```diff
@@ -397,14 +428,14 @@ class Document extends EventEmitter {
    *
    * @returns {true} Always true.
    */
-  isRoot() {
+  isRoot(): true {
```

Comment:
I think if we do this, this might allow us to remove some of the type assertions in the code that follows

```suggestion
  isRoot(): this is Document {
```

Reply:
Yeah, a little surprising that this works and `true` doesn’t :slightly_smiling_face:  Done!